boards/rp-pico/examples/pico_hd44780_display.rs

```diff
@@ -0,0 +1,137 @@
+//! # LCD Display Example
+//!
+//! In this example, the RP2040 is configured to drive a small two-line
+//! alphanumeric LCD using the
+//! [HD44780](https://crates.io/crates/hd44780-driver) driver.
+//!
+//! It drives the LCD by pushing data out of six GPIO pins. This version has
+//! been adapted from rp-hal/pr2040-hal/examples/lcd_display.rs for 
+//! the rp-pico board layout and pin assignment.
+//!
+//! See the `Cargo.toml` file for Copyright and license details.
+//!
+//! ```text              /--------------------------------------\
+//!  ____________        |        /-------------------------\   |
+//! | 1       GND|-------+---\    |           _|USB|_       |   |
+//! | 2       VDD|-------/   |    |          |1  R 40|-VBUS-o   v
+//! | 3        VS|-----------+----/          |2  P 39|       ||POT||
```

Comment:
Are you sure about this wiring?
AFAIK, pin 2 (VDD) should be the fixed voltage, and pin 3 (often called V0) should be an adjustable value between GND and VDD. So it looks like you switched pins 2 and 3 on the display connector.